### PR TITLE
fix(refine-log-filtering) Filter out specific logs #WPB-11179

### PR DESCRIPTION
**Pull request by spoonman01**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Logging to be improved in prod

### Solutions

Merge previously merged and deployed solution from staging branch.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: src/main/kotlin/com/wire/bots/polls/dto/roman/Message.kt [hunk 1]
```diff
@@ -97,7 +103,18 @@ data class Message(
...
+        override fun toString(): String {
+            return "PollObjectMessage(id='$id', buttons=$buttons, offset=$offset)"
+        }
+    }
+
+    /**
+     * Avoid printing out the token by mistake if object is printed.
+     */
+    override fun toString(): String {
+        return "Message(botId='$botId', userId=$userId, conversationId=$conversationId, type='$type', messageId=$messageId, text=$text, refMessageId=$refMessageId, reaction=$reaction, image=$image, handle=$handle, locale=$locale, poll=$poll, mimeType=$mimeType)"
```
Comment (lwille) [inline src/main/kotlin/com/wire/bots/polls/dto/roman/Message.kt:116]: **thought:** is this the default log format for this project? If not, could we use something that makes it easier to parse logs in the future?

**suggestion:** please use one of the following formats in order to help with log parsing:

* [logfmt](https://brandur.org/logfmt)
* json

Comment (spoonman01) [reply src/main/kotlin/com/wire/bots/polls/dto/roman/Message.kt:116]: All the log messages are already in JSON, this is configured project wise. Each log has a bunch of json fields from metadata, timestamp and the actual "message".
The messages are not json based, each logger can write any string they want. In this case, this is just the standard way objects are printed in Java

**Review by alexandreferris — APPROVED** [review]
Kotlin side looks good 👍🏻

**Review by jschumacher-wire — APPROVED** [review]
Github actions part: LGTM

**Pull request merged**